Client/src/Views/Question/DefaultQuestionForm.tsx

```diff
@@ -144,94 +145,92 @@ export default function DefaultQuestionForm(props: Props) {
   let renderParamGroup = props.renderParamGroup ? props.renderParamGroup : renderDefaultParamGroup;
   let Description = props.DescriptionComponent || QuestionDescription;
 
-  let fullContainerClassName = `${containerClassName || ''} ` + (
-    question.parameters.every(({ type }) => type !== 'filter') 
-      ? cx('', 'default-width')
-      : cx('', 'wide-width')
-  );
+  let fullContainerClassName = `${containerClassName || ''} ` + cx();
 
   let containerRef = React.useRef<HTMLDivElement>(null);
 
-  let [ navigatingToDescription, setNavigatingToDescription ] = React.useState(false);
-
-  let defaultOnClickDescriptionLink = React.useCallback((e: React.MouseEvent<HTMLAnchorElement>) => {
-    e.preventDefault();
-    const descriptionSection = containerRef.current && containerRef.current.querySelector(`.${cx('DescriptionSection')}`);
-
-    if (descriptionSection instanceof HTMLElement) {
-      scrollIntoView(descriptionSection);
-
-      setNavigatingToDescription(true);
-      setTimeout(() => {
-        setNavigatingToDescription(false);
-      }, 3000);
-    }
-  }, []);
-
-  let onClickDescriptionLink = props.onClickDescriptionLink || defaultOnClickDescriptionLink;
-
   React.useEffect(() => {
     if (stepValidation && !stepValidation.isValid && containerRef.current instanceof HTMLElement) {
       scrollIntoView(containerRef.current);
     }
-  }, [ stepValidation ]);
+  }, [stepValidation]);
```

Comment:
This isn't related to your code nor should we fix it in this PR, but it looks like some repos need to update their eslint setup to include the rules about React hooks (I'm thinking of exhaustive deps here).

https://reactjs.org/docs/hooks-rules.html.